core/controllers/contributor_dashboard_test.py

```diff
@@ -218,7 +218,7 @@ def test_get_skill_opportunity_data_does_not_throw_for_deleted_topics(self):
     def test_get_translation_opportunity_data(self):
```

Comment:
@sagangwee Could you update the test name to be clearer about what's being tested here (e.g. that all topics are being retrieved)?

In fact, shouldn't there be tests for what happens when the param is supplied vs when it isn't?

Reply:
Sure. Updated the tests.